.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,32 @@
+name: Pull Request
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: norio-nomura/action-swiftlint@3.2.1
+  test:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup
+        run: make setup
+        env:
```

Comment:
Really? I think that the step needs to set env values to use the secrets, don't you?

Reply:
Yes. The build and test will fail without the `environment.plist`. [It is ignored.](https://github.com/ca-dp/bucketeer-ios-sdk/blob/7769aaa8d4abe69613a5e90c0e8220a75fbc4a31/.gitignore#L24-L26).

Reply:
@jinSasaki 

I meant we don't need to pass the secrets to the environment variables.
If you run the `setup` with empty `API_URL` and `SDK_KEY`, the build and the unit will pass because those values are being used.

Also, for security reasons, there is a restriction in the GitHub Actions where people who send a PR from a forked project, the workflow won't trigger because of those secret values.

Reply:
OK!
Changed not to pass the secrets for the Pull Request workflow.
https://github.com/ca-dp/bucketeer-ios-sdk/pull/96/commits/00078c4698f3758436cd686af0d168535c0acf7c